doc/user-guide-v1.adoc

```diff
@@ -343,3 +343,10 @@ NOTE: The `RuntimeOperation` CR must be created in the same namespace as the Pod
 === Troubleshooting
 
 See the link:++troubleshooting.adoc++[troubleshooting guide] for information on how to investigate and resolve deployment problems.
+
+== Limitations
+=== Using certificates when manageTLS=true
+If `manageTLS=true` is specified in the CR YAML file, then the operator will manage the creation of a TLS secret for the runtime component.
```

Comment:
manageTLS is true by default - even if it's not specified in CR YAML. It'll be good to mention/clarify so that users don't remove it and expect manageTLS to be false